src/main/java/module/lineup/AufstellungsAssistentPanel.java

```diff
@@ -346,12 +346,10 @@ private void startAssistant(HOModel hoModel, HOMainFrame mainFrame) {
 			}
 		}
 
-		final Vector<Player> vPlayer = new Vector<Player>();
-		final Vector<Player> allePlayer = hoModel.getAllSpieler();
+		final List<Player> vPlayer = new Vector<Player>();
```

Comment:
I don't have full context on my phone, but shouldn't `Vector` be an `ArrayList` here too?

Reply:
confirmativ - i only changed the declaration as List, whereas i never changed the allocation. But i think it would be a good idea to change it to ArrayList